pipeline_live/data/sources/alpaca.py

```diff
@@ -6,8 +6,9 @@
 
 
 def list_symbols():
+    api = tradeapi.REST()
     return [
-        a.symbol for a in tradeapi.REST().list_assets()
+        a.symbol for a in api.list_assets()
```

Comment:
I guess this one is unnecessary for a single api call and was fine how it was before. I'm not sure if it the `with` was helping but it seemed like the connections were not being closed quickly and perhaps didn't until garbage collect or the program exited. The other option is to explicitly call `api.close()` when done.

Reply:
It looks like your setting up testing so this should show there because that's where my problem was but I've since moved on to other projects and wiped the env. Just wanted to share back a quick fix I found to eliminate some warning message noise.